kubernetes/configs/test/versions.sh

```diff
@@ -40,3 +40,4 @@ JOB_ANAC_CERTIFIED_ATTRIBUTES_IMPORTER_IMAGE_VERSION="1.0.23"
 JOB_IVASS_CERTIFIED_ATTRIBUTES_IMPORTER_IMAGE_VERSION="1.0.23"
 JOB_PN_CONSUMERS_IMAGE_VERSION="1.0.23"
 JOB_DTD_METRICS_IMAGE_VERSION="1.0.23"
+JOB_DATALAKE_DATA_EXPORT_IMAGE_VERSION="1.0.x-latest"
```

Comment:
is it ok to have the latest here?

Reply:
Yes, it's just a placeholder waiting for the 1.0.24 version.